draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1219,7 +1219,8 @@ before creating a new UUID. Distributed nodes are discussed in
 {{distributed_shared_knowledge}}.
 
 Implementations SHOULD choose one method for single-node UUID implementations
-that require batch UUID creation.
+that require batch UUID creation, or are otherwise concerned about monotonicity
+with high frequency UUID generation.
```

Comment:
This is the same edit from here https://github.com/ietf-wg-uuidrev/rfc4122bis/pull/79#discussion_r1161177270